exec/java-exec/src/main/java/org/apache/drill/exec/expr/fn/impl/NetworkFunctions.java

```diff
@@ -0,0 +1,619 @@
+
+    @Param
+    VarCharHolder inputTextA;
+
+    @Output BitHolder out;
+
+    @Inject
+    DrillBuf buffer;
+
+
+    public void setup() {
+    }
+
+
+    public void eval() {
+      String ipString = org.apache.drill.exec.expr.fn.impl.StringFunctionHelpers.toStringFromUTF8(inputTextA.start, inputTextA.end, inputTextA.buffer);
+
+      String[] ipAddressInArray = ipString.split("\\.");
+
+      int result = 0;
+
+      int[] octets = new int[3];
+
+      for( int i = 0; i < 3; i++ ){
+        octets[i] = Integer.parseInt( ipAddressInArray[i] );
+        if( octets[i] > 255 || octets[i] < 0 ) {
+          result = 0;
+        }
+      }
+
+      if( octets[0] == 192 && octets[1] == 168 ) {
+        result = 1;
+      }
+      else if (octets[0] == 172 && octets [1] >= 16 && octets[1] <= 31 ){
+        result = 1;
+      }
+      else if( octets[0] == 10 ) {
+        result = 1;
+      }
+      else {
+        result = 0;
+      }
+
+      out.value = result;
+    }
+  }
+  /**
+   *  This function converts an IPv4 address into a BigInt.  Useful for sorting IPs, or looking for IP ranges.
+   *  IE:
+   *  SELECT *
+   *  FROM <data>
+   *  ORDER BY inet_aton( ip ) ASC
+   */
+  @FunctionTemplate(
+    name = "inet_aton",
+    scope = FunctionTemplate.FunctionScope.SIMPLE,
+    nulls = FunctionTemplate.NullHandling.NULL_IF_NULL
+  )
+  public static class InetAtonFunction implements DrillSimpleFunc {
+
+    @Param
+    VarCharHolder inputTextA;
+
+    @Output BigIntHolder out;
+
+    @Inject
+    DrillBuf buffer;
+
+
+    public void setup() {
+    }
+
+
+    public void eval() {
+      String ipString = org.apache.drill.exec.expr.fn.impl.StringFunctionHelpers.toStringFromUTF8(inputTextA.start, inputTextA.end, inputTextA.buffer);
+      if( ipString == null || ipString.isEmpty() || ipString.length() == 0 ){
+        out.value = 0;
+      } else {
+        String[] ipAddressInArray = ipString.split("\\.");
+
+        long result = 0;
+        for (int i = 0; i < ipAddressInArray.length; i++) {
+          int power = 3 - i;
+          int ip = Integer.parseInt(ipAddressInArray[i]);
+          result += ip * Math.pow(256, power);
+        }
+
+        out.value = result;
+      }
+    }
+  }
+
+  /**
+   * Returns true if the input string is a valid IP address
+   */
+  @FunctionTemplate(
+    name = "is_valid_IP",
+    scope = FunctionTemplate.FunctionScope.SIMPLE,
+    nulls = FunctionTemplate.NullHandling.NULL_IF_NULL
+  )
+  public static class IsValidIPFunction implements DrillSimpleFunc {
+
+    @Param
+    VarCharHolder inputIP;
+
+    @Output BitHolder out;
+
+    @Inject
+    DrillBuf buffer;
+
+
+    public void setup() {
+    }
+
+
+    public void eval() {
+      String ipString = org.apache.drill.exec.expr.fn.impl.StringFunctionHelpers.toStringFromUTF8(inputIP.start, inputIP.end, inputIP.buffer);
+      if( ipString == null || ipString.isEmpty() || ipString.length() == 0 ){
```

Comment:
@cgivre regarding coding convention. I have mentioned this multiple time in your previous pull request and in this as well. You do fix some parts but still there are many places where code conventions are not followed. I suggest you follow guidelines from the link below [1], import formatter and format code using your IDE. It's really tedious for the code reviewer to point to all places where simple formatting are not observed which can be done by the developer with simple keyboard shortcut but following code conventions a common rule here and we all try to follow it!

[1] https://drill.apache.org/docs/apache-drill-contribution-guidelines/